<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>morebus</class>
 <widget class="QDialog" name="morebus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="4">
    <widget class="QLineEdit" name="get_zuoweishu"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="banci">
     <property name="text">
      <string>班次</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="get_banci"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="facheshijian">
     <property name="text">
      <string>发车时间</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="zuoweishu">
     <property name="text">
      <string>座位数</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="zhongdian">
     <property name="text">
      <string>终点</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="get_facheshijian"/>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="get_zhongdian"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
